app/models/metric/rollup.rb

```diff
@@ -152,6 +154,12 @@ def self.rollup_hourly(obj, hour, _interval_name, _time_profile, new_perf, orig_
     new_perf.reverse_merge!(orig_perf)
     new_perf.merge!(Metric::Processing.process_derived_columns(obj, new_perf, hour)) unless DERIVED_COLS_EXCLUDED_CLASSES.include?(obj.class.base_class.name)
 
+    # Add metric data collected from vmdb
+    # FIXME: use the process_derived_columns function
+    if orig_perf[:resource_type] == "ContainerNode"
+      new_perf[:container_group_creation_rate] = obj.container_groups.where(:ems_created_on => 1.hour.ago..Time.now.utc).count
```

Comment:
@yaacov I have the feeling that the interval (start - end) is passed here (in the context)